src/sage/combinat/permutation.py

```diff
@@ -1070,12 +1067,12 @@ def _to_cycles_set(self, singletons=True):
         cycles = []
 
         if not singletons:
-            #remove the fixed points
-            L = set(i+1 for i,pi in enumerate(p) if pi != i+1)
+            # remove the fixed points
+            L = {i+1 for i,pi in enumerate(p) if pi != i+1}
```

Comment:
I think there is a whitespace after the comma missing

Reply:
[PEP8](https://peps.python.org/pep-0008/#other-recommendations) says it's not required

> If operators with different priorities are used, consider adding whitespace around the operators with the lowest priority(ies). Use your own judgment; however, never use more than one space, and always have the same amount of whitespace on both sides of a binary operator:
```python
# Correct:
i = i + 1
submitted += 1
x = x*2 - 1
hypot2 = x*x + y*y
c = (a+b) * (a-b)
```

That's why I didn't point it out

Reply:
Comma, not operator.

Reply:
Sorry for rushing the review

Reply:
@fchapoton: in case you open a new ticket, please let me know.  (I admit I do not care enough, but I'm willing to browse through the changes.)